.github/workflows/feature_branch_sync.yml

```diff
@@ -0,0 +1,35 @@
+name: Rebase release_4.6-next and stable-2.6
```

Comment:
this workflow doesn't make sense in awx, and also I think it was removed in tower itself recently by @thedoubl3j

Reply:
it was yes. script was recently just removed.